test/server/server.test.ts

```diff
@@ -218,6 +236,52 @@ describe('Server', () => {
       expect(response.headers.location).toEqual('/graphiql');
     }
   );
+
+  it('returns reconciliation service manifest', async () => {
+    const response = await httpServer.inject({
+      method: 'GET',
+      url: '/reconciliation/https://data.netwerkdigitaalerfgoed.nl/rkd/rkdartists/sparql',
```

Comment:
I put the whole distribution IRI after `/reconciliation/` because it seems the service URL is not allowed to have parameters:

> When the reconciliation service [endpoint](https://reconciliation-api.github.io/specs/latest/#dfn-endpoint) is queried with a HTTP GET query without parameters, the [service manifest](https://reconciliation-api.github.io/specs/latest/#dfn-service-manifest) MUST be returned.

Am I right in interpreting this as _query_ parameters? Otherwise, `/reconciliation?iri=https://...` would be a possibility too.

Reply:
Make this `/reconcile/.…`